### DISCUSSION TITLE
vsphere-iso: dial tcp [ip_address]: connect: connection refused

### DISCUSSION THREAD

**Discussion by ghost** [hashicorp/packer-plugin-vsphere#635] [Community Help]
_This issue was originally opened by @raduad as hashicorp/packer#10222. It was migrated here as a result of the [Packer plugin split](https://github.com/hashicorp/packer/issues/8610#issuecomment-770034737). The original body of the issue is below._

<hr>

Getting connection refused during a vsphere-iso build.

Using
```
packer 1.6.5 / 1.6.3
provisioner used: ansible
Red Hat Enterprise Linux Server release 7.4 (Maipo)
Linux 3.10.0-862.11.6.el7.x86_64 hashicorp/packer#1 SMP Fri Aug 10 16:55:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

We see this during the provisioner phase of the build. It happens with both RHEL and SLES builds, but it happens sporadically. 

At the moment of the problem happening we can log in to the in-progress VM using SSH without any issues. There's no firewall setting, no condition blocking the connection.

**Comment by tenthirtyam** [comment]
For general questions, we recommend reaching out to the [community forum](https://discuss.hashicorp.com/c/packer) for greater visibility.  The GitHub issue tracker is only watched by a small group of maintainers and collaborators and is typically reserved for bug reports and enhancement request. You will typically have more success finding assistance on the forum.

If you find the forum to be more helpful or if you've found the answer to your question elsewhere, please comment and close the issue. If there is no further activity on this question within the next 30 days it will be closed.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.
